Changelog.md

```diff
@@ -1,3 +1,15 @@
+2.0.0-alpha1
+  - MAJOR: new `authPlugins` api replacing 
+    `authSwitchHandler`, added caching_sha2_password 
+    and mysql_native_password as default plugins.
+    Added tests for mysql 8 and ssl. Mysql 8 server
+    now supported with default settings.              #1021, #906, #991
+  - MAJOR: LOCAL INFILE does read from to local       #1034
+    fs now and requires `infileStreamFactory` option
```

Comment:
You missed the "not" part. How about:

```
  - MAJOR: LOCAL INFILE does not automatically read
    from the local fs and now requires an explicit
    `infileStreamFactory` to be specified in query options.
```

Reply:
yes, looks better ( unfortunately already merged this pr, will add to master )